modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -2379,6 +2379,125 @@ definition {
+ 
+ 			ObjectAdmin.openObjectAdmin();
+ 
+ 			ObjectPortlet.selectCustomObject(label = "Custom Object 185");
+ 
+ 			ObjectAdmin.goToActionsTab();
+ 
+ 			ObjectAdmin.addObjectActionViaUI(
+ 				actionLabel = "Action Label",
+ 				actionName = "ActionName",
+ 				active = "true",
+				errorMessage = "Error",
+ 				inputMethod = "true",
+ 				inputValue = "New Entry Test",
+ 				objectLabel = "Custom Object 185",
+ 				thenAction = "Add an Object Entry",
+ 				whenAction = "Standalone");
+ 
+ 			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+ 				fieldName = "customObjectField",
+ 				objectName = "CustomObject185",
+ 				value = "Entry Test");
+ 		}
+ 
+ 		task ("And: A new user is added") {
+ 			JSONUser.addUser(
+ 				userEmailAddress = "userea@liferay.com",
+ 				userFirstName = "userfn",
+ 				userLastName = "userln",
+ 				userScreenName = "usersn");
+ 		}
+ 
+ 		task ("When: The user has permissions to trigger a standalone action") {
+ 			JSONRole.addRegularRole(roleTitle = "Regular Role");
+ 
+ 			Role.openRolesAdmin();
+ 
+ 			LexiconEntry.gotoEntry(rowEntry = "Regular Role");
+ 
+ 			Navigator.gotoNavItem(navItem = "Define Permissions");
+ 
+			ObjectAdmin.searchInDefinePermissions(search = "Custom Objects 185");
+ 
+ 			Click(
+ 				key_wikiPageVersionNew = "action.ActionName",
+ 				locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+ 
+ 			PortletEntry.save();
+ 
+ 			var objectId = JSONObject.getObjectId(objectName = "CustomObject185");
+ 
+			Permissions.definePermissionViaJSONAPI(
+ 				resourceAction = "VIEW_CONTROL_PANEL",
+ 				resourceName = 90,
+ 				roleTitle = "Regular Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "ACCESS_IN_CONTROL_PANEL",
+				resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+				roleTitle = "Regular Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW",
+				resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+				roleTitle = "Regular Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW",
+				resourceName = "com.liferay.object.model.ObjectDefinition#${objectId}",
+				roleTitle = "Regular Role",
+				roleType = "regular");
+		}
+
+		task ("Then: User should be able to trigger the action in the drop-down list") {
+			JSONRole.assignRoleToUser(
+				roleTitle = "Regular Role",
+				userEmailAddress = "userea@liferay.com");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject185");
+
+			Click(
+				key_entry = "Entry Test",
+				locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+
+			Click(
+				key_dropdowntItem = "Action Label",
```

Comment:
Can you fix the typo in the path? It should be "key_dropdownItem" not "key_dropdowntItem"

You'll need to apply this change to every instance of this typo.